debian/copyright

```diff
@@ -0,0 +1,38 @@
+Format: http://www.debian.org/doc/packaging-manuals/copyright-format/1.0/
+Upstream-Name: coding-shell-extensions
+Source: <url://example.com>
+
+Files: *
+Copyright: <years> <put author's name and email here>
```

Comment:
@krnowak  yes not sure who the author will be? will it be endless or something else. Maybe something like they did here -> https://github.com/endlessm/coding-game-manager/blob/master/debian/copyright